rasa/core/policies/intent_ted_policy.py

```diff
@@ -211,8 +213,21 @@ class IntentTEDPolicy(TEDPolicy):
         # Specify what features to use as sequence and sentence features.
         # By default all features in the pipeline are used.
         FEATURIZERS: [],
-        # List of intents to ignore
+        # List of intents to ignore for `action_unlikely_intent` prediction.
         IGNORE_INTENTS_LIST: [],
+        # Tolerance for prediction of action_unlikely_intent.
+        # This is specified as a ratio of negative trackers for
+        # which the predicted similarity score of the
+        # corresponding correct label is above a threshold.
+        # Hence, any tracker at inference time
+        # should result in a predicted similarity score equal
+        # to or above the threshold in order to avoid triggering
+        # `action_unlikely_intent`. Higher values of `tolerance`
```

Comment:
I'm having trouble making sense of this description. Is the following technically correct and if so would it suffice as a description:
For each intent, the tolerance is the percentage of negative training instances that would be ignored by IntentTED. This is converted into a similarity threshold by identifying the similarity score for the (1 - tolerance) percentile of negative examples. Any tracker with a similarity score below this threshold will trigger an `action_unlikely_intent`.

Reply:
Thanks for the suggestion. I incorporated it in.

Reply:
Looks great!